spec/rspec/mocks/partial_double_spec.rb

```diff
@@ -379,6 +390,22 @@ def defined_private_method
         object.send(:defined_private_method)
       end
 
+      it 'runs the before_verifying_double callbacks on any_instance' do
+        expect { |probe|
+          RSpec.configuration.mock_with(:rspec) do |config|
+            config.before_verifying_doubles(&probe)
+          end
+
+          expect_any_instance_of(klass).to receive(:implemented)
+          object.implemented
+        }.to yield_successive_args(
+          *(klass.ancestors + [klass]).map do |target|
```

Comment:
I was initially struggling to trigger this only once but I've come up with a better solution that I'll push up today sometime